src/utils/type-utils.ts

```diff
@@ -16,12 +16,8 @@ export const _isUint8Array = function (x: unknown): x is Uint8Array {
 // fails on only one very rare and deliberate custom object:
```

Comment:
fun fact: the URL in this comment might have said something about JS in the past but it doesn't anymore 🤷